### PR TITLE
Feat/cs 1751 add task to queue

### PR DESCRIPTION
**Pull request by RamaChandiniDevarapalli**

### CODE REVIEW

**Comment by thdlam** [comment]
Looks good, you just have linting warnings that  you need to correct.

**Review by thdlam — APPROVED** [review]

**Review by athena-chen-chen — CHANGES_REQUESTED** [review]

**Comment by athena-chen-chen** [comment]
Please fix lint warning as well.

File: apps/tenant-management-webapp/src/app/pages/admin/services/task/taskModal.tsx [hunk 1]
```diff
@@ -0,0 +1,196 @@
+import React, { FunctionComponent, useState } from 'react';
+import {
+  GoAForm,
```
[the remark this answers is not in the document]
Comment (RamaChandiniDevarapalli) [reply apps/tenant-management-webapp/src/app/pages/admin/services/task/taskModal.tsx]: updated changes

File: apps/tenant-management-webapp/src/app/pages/admin/services/task/tasksList.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        <TaskListTable
+          tasks={tasks}
+          onEditTask={(updatedTask) => {
+            setModalType('edit');
+            setUpdatedTask(updatedTask);
+            setOpenAddTask(true);
+          }}
+        />
+      )}
+      {openAddTask && (
```
Comment (athena-chen-chen) [inline apps/tenant-management-webapp/src/app/pages/admin/services/task/tasksList.tsx:127]: I am afraid here will have future bug.

Comment (athena-chen-chen) [reply apps/tenant-management-webapp/src/app/pages/admin/services/task/tasksList.tsx:127]: Remove lin 127 and change open = {openAddTask}

File: apps/tenant-management-webapp/src/app/pages/admin/services/task/taskModal.tsx [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+        </GoAButtonGroup>
+      }
+    >
+      <div>
+        <GoAFormItem error={errors?.['name']} label="Name">
+          <GoAInput
+            type="text"
+            name="name"
+            value={task?.name}
+            data-testid={`task-modal-name-input`}
```
Comment (athena-chen-chen) [inline apps/tenant-management-webapp/src/app/pages/admin/services/task/taskModal.tsx:134]: change to testId

**Pull request merged**